README.md

```diff
@@ -72,6 +72,10 @@ REACT_APP_APP_ID=<YOUR_FIREBASE_API_ID>
 REACT_APP_BACKEND_URL=http://localhost:5000
 ```
 
+### Connect Backend to Project
+
+To be able to use `firebase-admin`, select the gear icon next to the Project Overview screen, go to **Project Settings** > **Service Accounts** and generate a new private key. Name the file `serviceAccountKey.json` and paste the JSON file in the "backend" folder. Then add `/backend/serviceAccountKey.json` to your `.gitignore`.
```

Comment:
use a numbered list like the other instructions to break down the steps. remove the last step, the gitignore will be updated for all devs once this PR is merged